dome9CloudBots/bots/storage_account_enable_https_traffic_only.py

```diff
@@ -0,0 +1,34 @@
+# What it does: enable Storage account Secure transfer required field
+# Usage: AUTO: storage_account_enable_https_traffic_only
+# Limitations: None
+
+import logging
+from msrestazure.azure_exceptions import CloudError
+from azure.mgmt.storage import StorageManagementClient
+from azure.mgmt.storage.models import StorageAccountUpdateParameters
+
+
+def run_action(credentials, rule, entity, params):
+    logging.info(f'{__file__} - run_action started')
+    subscription_id = entity.get('accountNumber')
+    group_name = entity.get('resourceGroup', {}).get('name')
```

Comment:
Same as the above. Just use `entity['resourceGroup']['name']`. It doesn't exists? so an exception will be thrown and that is what we want.